src/components/KeyboardAwareScrollView/KeyboardAwareBase.js

```diff
@@ -146,12 +146,16 @@ export default class KeyboardAwareBase extends Component {
         this._keyboardAwareView.contentSize.height -
         this._keyboardAwareView.layout.height +
         this._keyboardAwareView.props.contentInset.bottom;
-      this._keyboardAwareView.scrollTo({x: 0, y: bottomYOffset, animated: scrollAnimated});
+      this.scrollTo({x: 0, y: bottomYOffset, animated: scrollAnimated});
     }
   }
   scrollTo(options) {
     if (this._keyboardAwareView) {
-      this._keyboardAwareView.scrollTo(options);
+      if (this._keyboardAwareView.scrollTo) {
+        this._keyboardAwareView.scrollTo(options);
+      } else if (this._keyboardAwareView.scrollToOffset) { // scroll to start in FlatList
+        this._keyboardAwareView.scrollToOffset({...options, offset: 0});
+      }
```

Comment:
Why in the case of FlatList you scroll to offset: 0? 
Shouldn't you scroll to the y offset passed in options?

Reply:
Fixed, some logic has changed and Android should also work now.